libraries/AP_LightWareSerial/AP_LightWareSerial.cpp

```diff
@@ -13,72 +13,72 @@
    along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "AP_Proximity_LightWareSerial.h"
+#include "AP_LightWareSerial.h"
+
+#if AP_PROXIMITY_LIGHTWARE_SF45B_ENABLED ||  AP_RANGEFINDER_LIGHTWARE_GRF_ENABLED
```

Comment:
Please create an `AP_LightWareSerial_config.h` and a new `AP_LIGHTWARESERIAL_ENABLED` define (set to `(AP_PROXIMITY_LIGHTWARE_SF45B_ENABLED ||  AP_RANGEFINDER_LIGHTWARE_GRF_ENABLED)`)